core/worker.go

```diff
@@ -899,7 +1069,7 @@ func (w *worker) commitTransaction(env *environment, parent *types.WorkObject, t
 
 var qiTxErrs uint64
 
-func (w *worker) commitTransactions(env *environment, parent *types.WorkObject, txs *types.TransactionsByPriceAndNonce, interrupt *int32) error {
+func (w *worker) commitTransactions(env *environment, primeTerminus *types.WorkObjectHeader, parent *types.WorkObject, txs *types.TransactionsByPriceAndNonce, interrupt *int32, excludeEtx bool) error {
```

Comment:
Actually I just realized the interrupt will break this PR as well because it will interrupt before you include the lowest fee tx so the basefee in the header will be wrong

Reply:
Agreed, removed it